knowledge-base/configure-fiddler-on-fedora.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Configure Fiddler Everywhere on Fedora, CentOS, or RedHat
+description: Instructions for configuring Fiddler Everywhere trust certificate on Fedora, CentOS, RedHat, or any other Linux distribution that uses shared system certificates storage.
+type: how-to
+slug: fiddler-fedora-centos
+tags: Fiddler shared system certificates storage, Fiddler Everywhere Fedora, Fiddler Everywhere Red Hat, Fiddler Everywhere Xubuntu, Fiddler CentOS
+publish: true
+res_type: kb
+---
+
+
+#### Environment
+
+|   |   |
+|---|---|
+| Product  | Fiddler Everywhere  |
+| Product Version | 1.0.0 and above  |
+| 3rd-party tool | [Shared System Certificates storage](https://docs.fedoraproject.org/en-US/quick-docs/using-shared-system-certificates/) |
+| Supported OS | Fedora, CentOS, RedHat, and others |
+
+
+#### Description
+
+The Fiddler Everywhere [instructions for installing the root trust certificate for Linux]({%slug configuration-get-started%}#configure-on-linux) are based on using the Debian **dkpg-reconfigure** command. However, some Linux distributions like Fedora, CentOS, RedHat, and others might lack the dkpg library or limit its usage. In this case, you will need to install and trust the Fiddler root certificate while using the **update-ca-trust** command as shown in the solution below.
+
+
+## Solution
+
+1. Install and open Fiddler Everywhere application on Fedora, CentOS, RedHat, or other Linux distribution. In case the distribution provides the dkpg-reconfigure command, proceed with [the standard instructions]({%slug configuration-get-started%}#configure-on-linux). Otherwise, proceed to the next step.
```

Comment:
```suggestion
1. Install and open the Fiddler Everywhere application on Fedora, CentOS, RedHat, or other Linux distribution. In case the distribution provides the dkpg-reconfigure command, proceed with [the standard instructions]({%slug configuration-get-started%}#configure-on-linux). Otherwise, proceed to the next step.
```